packages/amplify-category-hosting/lib/S3AndCloudFront/index.js

```diff
@@ -53,22 +48,7 @@ async function enable(context) {
 }
 
 async function checkCDN(context) {
```

Comment:
we can actually move that function entirely. I would suggest, since buckets don't have context.exeInfo.template.Resources.S3Bucket.Properties.AccessControl by default, we don't need to delete it anymore.

Reply:
That makes sense. Just make sure you do an exhaustive search in the codebase and remove all references